com.endlessm.CompanionAppService.json.in

```diff
@@ -24,6 +24,37 @@
     ],
     "modules": [
         "com.endlessm.CompanionAppService.PipDependencies.json",
+        {
+            "name": "basin",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/endlessm/basin",
+                    "branch": "T21272-patch"
```

Comment:
AIUI, @tchx84  intends to rework things to avoid the confusion. For now I'd like to avoid more churn up there since we're going to eliminate our usage of basin in T20853 anyway.

Reply:
OK... not a fan of shipping something from a branch like this, but if it's only temporary, maybe that's fine.